src/Sentry.Unity.Editor/Native/SentryWindowsPlayer.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using Sentry.Extensibility;
+using UnityEditor;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace Sentry.Unity.Editor.Native
+{
+    internal static class SentryWindowsPlayer
+    {
+        internal static readonly ProcessStartInfo StartInfo = new()
```

Comment:
It's best to allocate this when needed and let GC collect it. Instantiating this is cheap: https://github.com/dotnet/runtime/blob/78e840e1072127d68714b55cffdcfa96a7e2f8af/src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs#L20-L36